<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Login</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="ID">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PW">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Are you beginner?</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>69</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>PW</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>60</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign In</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LOGIN FOR USING OUR SERVICE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>130</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
     <widget class="QLabel" name="DEBUG_RESULT">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>SignUp</string>
     </attribute>
     <widget class="QLabel" name="DEBUG_RESULT_IS_SUCCEED">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>140</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="ID_new">
         <property name="text">
          <string>(type your new id)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PW_new">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string>(type your new password)</string>
         </property>
         <property name="maxLength">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>91</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>PW</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>60</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>WELCOME! TYPE YOUR NEW ID AND PASSWORD</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>63</y>
        <width>67</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>SongList</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>121</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>PLAY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string>DELETE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>name1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>name2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>name3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>name4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>name5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>name6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>name7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>name8</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>SONG LIST</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>MYLIST</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>MyList</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>name1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>name2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>name3</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>121</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>RECORD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>UPLOAD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>RENAME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>DELETE</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>MY LIST</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Register</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>191</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLineEdit" name="ARTIST">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SONG_NAME">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_9">
      <property name="geometry">
       <rect>
        <x>158</x>
        <y>80</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>ARTIST</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>SONG NAME</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolButton" name="toolButton_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>90</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>REGISTER</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TYPE YOUR ARTIST AND SONG NAME</string>
      </property>
     </widget>
     <widget class="QLabel" name="DEBUG_REGISTER">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>291</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
